### PR TITLE
Update KEB docs

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Update the info about the trial period in Runtime Operations.
- Fix links to the initialization stages in Runtime Operations.
- Update the info on the swagger.
- Fix typos.

**Related issue(s)**

### CODE REVIEW

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The upgrade process contains the following steps:
 
 ## Provide additional steps
 
-You can configure Runtime operations by providing additional steps. To add a new step, follow these tutorials:
+You can configure Runtime operations by providing additional steps. To add a new step, follow the following tutorials:
```
Comment (majakurcius) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:99]:
```suggestion
You can configure Runtime operations by providing additional steps. To add a new step, follow the following tutorials:
```
`Follow the following` sounds a tad weird. Can we rephrase that?

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 2]
```diff
@@ -10,10 +10,13 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
 
 ## Use Virtual Service
 
-Open the following website:
+Open one of the following websites:
 
    ```
```
Comment (majakurcius) [inline docs/kyma-environment-broker/03-11-swagger.md:15]:
```suggestion
```
If the next suggestion is applied, this has to be removed.

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 3]
```diff
@@ -10,10 +10,13 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
...
-   https://$BROKER_URL/
+   https://kyma-env-broker.cp.dev.kyma.cloud.sap/
+   ```
+   ```
+   https://kyma-env-broker.cp.stage.kyma.cloud.sap/
    ```
```
Comment (majakurcius) [inline docs/kyma-environment-broker/03-11-swagger.md:20]:
```suggestion
- [`https://kyma-env-broker.cp.dev.kyma.cloud.sap/`](https://kyma-env-broker.cp.dev.kyma.cloud.sap/)
- [`https://kyma-env-broker.cp.stage.kyma.cloud.sap/`](https://kyma-env-broker.cp.stage.kyma.cloud.sap/)
```
There's no need to format these as full code blocks, these are just values that we list. We can, however, link them so you don't need to copy and paste them, and just click to open.

**Review by majakurcius — COMMENTED** [review]
Good job. I can really see the adherence to our guidelines here! :)

File: docs/kyma-environment-broker/03-03-runtime-operations.md (hunk 1 above)
Comment (IwonaLanger) [reply docs/kyma-environment-broker/03-03-runtime-operations.md:99]: Sure, somehow I've overlooked that.

Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:99]:
```suggestion
You can configure Runtime operations by providing additional steps. To add a new step, follow the tutorials:
```

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 4]
```diff
@@ -10,10 +10,12 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
 
 ## Use Virtual Service
 
-Open the following website:
+Open one of the following websites:
 
+   https://kyma-env-broker.cp.dev.kyma.cloud.sap/
+   ```
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-11-swagger.md:16]:
```suggestion
Open Kyma Environment Broker APIs on [development env](https://kyma-env-broker.cp.dev.kyma.cloud.sap/) or on [stage env](https://kyma-env-broker.cp.stage.kyma.cloud.sap/).

```

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 5]
```diff
@@ -10,10 +10,12 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
...
    ```
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-11-swagger.md:17]:
```suggestion
  
```

File: docs/kyma-environment-broker/03-11-swagger.md [hunk 6]
```diff
@@ -10,10 +10,12 @@ You can either use Virtual Service or port-forward the Pod to expose and use the
...
-   https://$BROKER_URL/
+   https://kyma-env-broker.cp.stage.kyma.cloud.sap/
    ```
```
Comment (IwonaLanger) [inline docs/kyma-environment-broker/03-11-swagger.md:19]:
```suggestion
```

File: docs/kyma-environment-broker/03-03-runtime-operations.md (hunk 1 above)
Comment (majakurcius) [reply docs/kyma-environment-broker/03-03-runtime-operations.md]: Maybe `these` would be an even better choice?

File: docs/kyma-environment-broker/03-03-runtime-operations.md [hunk 7]
```diff
@@ -96,7 +96,7 @@ The upgrade process contains the following steps:
 
 ## Provide additional steps
 
-You can configure Runtime operations by providing additional steps. To add a new step, follow these tutorials:
+You can configure Runtime operations by providing additional steps. To add a new step, follow the tutorials:
```
Comment (majakurcius) [inline docs/kyma-environment-broker/03-03-runtime-operations.md:99]:
```suggestion
You can configure Runtime operations by providing additional steps. To add a new step, follow these tutorials:
```
I'd stick to `these`. What's wrong with that?

**Review by majakurcius — APPROVED** [review]

**Pull request merged**